frontend/components/molecules/Form/FieldSection.js

```diff
@@ -27,15 +27,36 @@ const DescriptionSection = ({ text }) => {
 const FieldSection = ({
     title,
     description = '',
-    type = 'text',
+    type,
     required,
     onChange,
     name,
     value,
     hasError = false,
-    errorText = 'Incorrect answer provided. Please double-check your answer.',
+    errorText,
     options = [],
 }) => {
+    // checks for errorText first if its undefined. If it is, then figure out the type and set the errorMessage based off the type. If errorText is defined
+    // then no need to check for type, it has a specific message from backend
+    if (typeof errorText === 'undefined') {
+        switch (type) {
+            case 'email':
+                errorText = 'Please enter a valid email.';
+                break;
+            case 'phone':
+                errorText = 'Please enter a valid 10 digit phone number.';
+                break;
+            case 'text':
+                errorText = 'Please provide a response to this question.';
+                break;
+            case 'radio':
+                errorText = 'Please select an option above.';
+                break;
+            case 'boolean':
+                errorText = 'Please select an option above.';
+                break;
+        }
+    }
```

Comment:
You can do something like this instead:

`const errorMsg = errorText ?? getDefaultErrorText(type);`

Where getDefaultErrorText is:

```
function getDefaultErrorText(type) {
    switch (type) {
        case 'text':
        case 'numbers': 
        ...
    }
}
```

And please move getDefaultErrorText into `frontend/util.js` file.

Reply:
Once you finish refactoring, can you please tell me what the first statement I wrote does ? Thanks.

Reply:
The first statement sets the value of errorMsg to getDefaultErrorText if errorText is null, so that it can run the switch case in the util file and set the error text. Then, it displays errorMsg under the form. If errorText already has a value from the backend, the errorMsg will just be errorText